modules/nodes-secondary-scheduler-about.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/index.adoc
+
+:_content-type: CONCEPT
+[id="nodes-secondary-scheduler-about_{context}"]
+= About the {secondary-scheduler-operator}
+
+The {secondary-scheduler-operator-full} provides a way to deploy a custom secondary scheduler in {product-title}. The secondary scheduler runs alongside the default scheduler to schedule pods. Pod configurations can specify which scheduler to use.
+
+The custom scheduler must have the `/bin/kube-scheduler` binary and be based on the link:https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/[Kubernetes scheduling framework].
+
+[IMPORTANT]
+====
+You can use the {secondary-scheduler-operator} to deploy a custom secondary scheduler in {product-title}, but Red Hat does not directly support the functionality of the custom scheduler.
+
+For information on how to configure the default scheduler, see _Controlling pod placement using the scheduler_.
+====
```

Comment:
@bergerhoffer any specific reason we are talking about default scheduler here ? I think it may not be necessary right ?

Reply:
I had pulled this wording from a similar note we had in the existing custom scheduler docs. I think that the intention was to say hey you can do a custom scheduler. But if you just want to tweak the existing default scheduler instead, go see this. But I'll go ahead and remove this sentence if it might do more harm than good.